src/js/pages/jobs/JobDetailPage.js

```diff
@@ -242,12 +245,10 @@ class JobDetailPage extends mixin(TabsMixin) {
 
     return (
       <Page>
-        <JobsMenu
+        <Page.Header
```

Comment:
This is the juice of the change. To keep the Page.Header and not create a new component with the new data layer. Please tell me if you like this way better.

Reply:
I agree with that, when we started I have not seem the problem it would cause to Page.Header. I agree we should not touch it, I like the passing an array solution.